app/src/main/res/layout/activity_main.xml

```diff
@@ -65,4 +75,13 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="@id/home_page_frame_content" />
 
+    <FrameLayout
+        android:id="@+id/screen_layout"
```

Comment:
This could have a better name...
something like search_layout_placeholder.
And you should keep it invisible while the user is not in the search flow.